maps/adhomai/code/machinery/phonograph.dm

```diff
@@ -0,0 +1,149 @@
+datum/track/New(var/title_name, var/audio)
+	title = title_name
+	sound = audio
+
+/obj/machinery/media/phonograph
```

Comment:
I am sure that this could just be a subtype of the /obj/machinery/media/jukebox instead of being just copy paste of the code with some minor alterations.

Reply:
That was the idea, but this was changed to require no power, which involved altering a lot of flags.

Reply:
Working on a change.